ipgeo/leo_test.go

```
@@ -0,0 +1,78 @@
+	"os"
+	"sync"
+	"testing"
+	"time"
+
+	"github.com/nxtrace/NTrace-core/wshandle"
+)
+
+func TestLeoIPWaitsForConnectionBeforeSending(t *testing.T) {
+	oldGet := getLeoWsConn
+	oldPools := IPPools.pool
+	oldOnce := receiveParseOnce
```

golangci-lint (2.11.4)

[error] copylocks: assignment copies lock value to oldOnce: sync.Once contains sync.noCopy

(govet)